sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoEquityOption.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.tutorials.Instruments
+{
+    [TestFixture]
+    public class DemoEquityOption
+    {
+        private ITransactionPortfoliosApi _transactionPortfoliosApi;
+        private IInstrumentsApi _instrumentsApi;
+        private IQuotesApi _quotesApi;
+        private IComplexMarketDataApi _complexMarketDataApi;
+        private IConfigurationRecipeApi _recipeApi;
+        private IAggregationApi _aggregationApi;
+
+        private TestDataUtilities _testDataUtilities;
+        private readonly DateTimeOffset _demoEffectiveAt = new DateTimeOffset(2020, 7, 1, 0, 0, 0, TimeSpan.Zero);
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            // if we are just demoing creation, then all we need is the instruments api
+            var apiFactory = TestLusidApiFactoryBuilder.CreateApiFactory("secrets.json");
+
+            _transactionPortfoliosApi = apiFactory.Api<ITransactionPortfoliosApi>();
+            _instrumentsApi = apiFactory.Api<IInstrumentsApi>();
+            _quotesApi = apiFactory.Api<IQuotesApi>();
+            _complexMarketDataApi = apiFactory.Api<IComplexMarketDataApi>();
+            _recipeApi = apiFactory.Api<IConfigurationRecipeApi>();
+            _aggregationApi = apiFactory.Api<IAggregationApi>();
+
+            _testDataUtilities = new TestDataUtilities(_transactionPortfoliosApi,
```

Comment:
would it be cleaner to pass in the apiFactory into the TestDataUtilities and create all the sub-apis inside?